app/views/content/blog/my-first-week-as-a-trainee-teacher.md

```diff
@@ -0,0 +1,69 @@
+---
+title: My first week as a trainee teacher
+author: Cindy Ventura
+date: "2021-09-17"
+images:
+  cindy-ventura:
+    path: "media/images/content/blog/cindy-ventura.jpg"
+    alt: "A photograph of trainee teacher Cindy Ventura"
+description: |-
+  Cindy recently started the Postgraduate Diploma in Education (PGDE) teacher training course at Sheffield University. She kept a diary during her first week on the course and shares her experiences here.
+keywords:
+  - teacher training
+  - trainee teacher course
+  - university-led teacher training
+tags:
+  - becoming a teacher
+---
+
+$cindy-ventura$
+
+Cindy recently started the Postgraduate Diploma in Education (PGDE) teacher training course at Sheffield University. She kept a diary during her first week on the course and shares her experiences here.
+
```

Comment:
Is it worth putting a subheading here to break up the intro from Cindy's words?

Reply:
How about Cindy's diary?

Reply:
Good call. On other posts we make the final paragraph italic so it's clear it's separate.


I created [a ticket on trello](https://trello.com/c/sqqWfKfn/1701-improve-blog-footers) on the subject of improving the footers in general.